docs/DB.md

```diff
@@ -69,3 +69,17 @@ Install postgis extension:
 ```psql
 create extension postgis;
 ```
+
+### CrunchyDB Openshift Cluster
+
+The database is deployed using the CrunchyDB postgres operator: https://github.com/CrunchyData/postgres-operator
+
+#### Standby Cluster Restore
+
+To spin up a standby cluster that bootstraps itself from our configured pgbackrest repo run:
+
+`PROJ_TARGET=<your-namespace> BUCKET=<your-bucket> bash openshift/scripts/oc_provision_crunchy_standby.sh <your-suffix> apply`
```

Comment:
Is PROJ_TARGET the OpenShift license plate that we want to deploy to?
What would `<your-suffix>` be for a prod deployment?

Reply:
Yep, it would be `prod`